script/jenkins/linux-test.sh

```diff
@@ -11,16 +11,6 @@ jenkins_setup
 #}
 #trap cleanup EXIT
 
-make test-deps testsuite
-
-export COV_DIR="build/coverage"
-
-for i in "full.cov" "integration.cov" "system.cov" "unit.cov"
-do
-  name=$(basename ${i} .cov)
-  if [ -f "${COV_DIR}/${i}" ]; then 
-    go tool cover -html="${COV_DIR}/${i}" -o "${COV_DIR}/coverage-${name}-report.html"
-    gocover-cobertura < "${COV_DIR}/${i}" > "${COV_DIR}/coverage-${name}-report.xml"
-  fi
-done
```

Comment:
Maybe I overlooked it, but I couldn't find a coverage stage or report anymore with this being removed.

Reply:
we don't have (and never had) any coverage for python tests, do we? 
we should still have them for unit tests, they are generated in the corresponding stage.

Reply:
I am probably just blind, but I really can't find where the code coverage for the unit tests should be living now.

Reply:
Ok, where do you look them up? I confess I never been interested in them, and I was expecting to be generated here: https://github.com/elastic/apm-server/blob/master/script/jenkins/unit-test.sh

But now that you mention it, I don't see any `go tool cover` there, just the test run report. So if that it is important I think that is the right place. Makes sense?

Reply:
Yes I agree that the unit tests would be the right place to create the coverage report now. I don't look at them quite often, but sometimes on a PR the code coverage stage failed, and I assumed that was done by the now removed code. 
If you look at the last stage of the `system tests`, e.g. [here](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-server%2Fapm-server-mbp/detail/PR-2621/2/pipeline/156) it gives you a link to navigate to the code coverage overview.

Reply:
Good point, thanks.

Reply:
fixed now https://apm-ci.elastic.co/job/apm-server/job/apm-server-mbp/job/PR-2565/30/Coverage-Sourcecode-Files/